about.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>About page</title>
+</head>
+<body>
+    <section class="sec1">
+        <div class="header">
+            <button class="btn1">
+                <img id="show-mobile-menu" class="humburger-icon" src="./img/mobile_menu_icon.png" alt="union icon">
+            </button>
+            <div id="mobile-menue"></div>
+            
+        </div>
+        <div class="black-strip ">
+           
+            <nav class="top-menu-navbar">
+                <ul class="navbar-list">
+                    <li class="navbar-item">
+                        <a href="https://www.facebook.com/">
+                            <img src="./img/facebook.png" alt="facebook">
+                        </a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="https://twitter.com/">
+                            <img src="./img/twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#">
+                            English
+                        </a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#">
+                            My Page
+                        </a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#">
+                            Logout
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="white-strap">
+            <div class="icon-wrapper">
+                <a href="./index.html">
+                    <img class="icon-img" src="./img/software.png" alt="website icon">
+                </a>
+            </div>
+            <nav class="top-menu-white-navbar">
+                <ul class="navbar-list-white">
+                    <li class="navbar-item-white ">
+                        <a href="./index.html">Home</a> 
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">News</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Sponser</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Join</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Program</a>
```

Comment:
- [ optional] you can add links to different page sections so as to ease navigation